projects/default/controllers/ros/RosSupervisor.cpp

```diff
@@ -449,6 +449,7 @@ bool RosSupervisor::virtualRealityHeadsetIsUsedCallback(webots_ros::get_bool::Re
   return true;
 }
 
+// cppcheck-suppress constParameter
```

Comment:
It is strange that the warning appears only to those few functions and not all the others. Do you have an explanation for this?

Reply:
No idea. I guess it is unable to detect that in some case the *Request* object can be modified. Maybe this *Request* has some non-standard method modifying it?

Reply:
> Maybe this Request has some non-standard method modifying it?

No idea what the ROS libraries are doing behind, but what looks strange is that it works exactly the same way as for the other callback so I was wondering why the error was raised only here. But that was just by curiosity.

Reply:
It's not always the very same *Request* object (different namespace), that might explain why it behaves differently.